### ISSUE TITLE
Relationship field with custom post statuses doesn't save

### ISSUE THREAD

**Issue by rosalynpoort** [pods-framework/pods#4437]
I have an advanced content type pod with a relationship field that relates to a custom post type. i have custom post statuses for the custom post type. on the relationship field in the advanced content type pod i can choose to display the posts with my custom statuses but the field won't save. it only saves when i choose a published post

**Comment by jimtrue** [comment]
@pglewis Need to verify this is an issue on 2.7 pick fields. I'm testing that today. If it's corrected in 2.7 Yay! But if we give the option to allow relationships on specific post statuses, we should be saving the connection.

**Comment by jimtrue** [comment]
@rosalynpoort What are the custom post statuses you have that you're trying to assign relationships to? I tested with 'draft' and while the Draft items showed up in the pick list, they wouldn't save to the relationship field.

**Comment by rosalynpoort** [comment]
@jimtrue yes i tested with 'draft' as well as 'pending-review' and 'maybe'. same results, they all show up in the pick list but won't save to the field

**Comment by jimtrue** [comment]
I just tested in the beta 2.7 as well and while it allows me to add a new draft relationship post and it adds it to the list. When I save, it's no longer in the related record post. This is most definitely a bug.

**Comment by rosalynpoort** [comment]
@jimtrue thanks for looking into it. I've worked around it for now by just making it a text field for the ID as I'm adding using the pods api anyway and not the WP dashboard.

**Comment by jimtrue** [comment]
For clarification: this is the 'Post Status' under Additional Options. This provides what posts show up in the Relationship pick list. Changing the status allows them to show, but they aren't saving to the field on save.

<img width="706" alt="screenshot 2017-08-30 12 50 46" src="https://user-images.githubusercontent.com/3052875/29884648-18a0bdd6-8d82-11e7-86d6-0d329dafeede.png">

**Comment by jamesgol** [comment]
Looking at the database it looks like the data is actually being saved, but just never loaded to be displayed.  I'm closing in on the problem.

**Comment by th-santos** [comment]
I am facing the same issue.

I have an Extended Post Type ("post") with a "Relationship" Field Type (non-bidirectional) related to a Custom Post Type ("company"). 

I am using `pods_api_post_save_pod_item_post` to get field values from my "company" post type. If "company" status is "publish", I can get those values, but if it is "private", I cannot get any field value.
